apps/cow-fi/modules/analytics/events.ts

```diff
@@ -0,0 +1,110 @@
+import { initCowAnalyticsGoogle } from '@cowprotocol/analytics'
+
+export const cowAnalytics = initCowAnalyticsGoogle()
+
+export enum Category {
```

Comment:
Now, there's one enum `Category` per app. Before apps were sharing the same categories and there was some categories not part of the enum